smoke-tests/apps/SamplingOverrides/src/smokeTest/resources/applicationinsights3.json

```diff
@@ -0,0 +1,39 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100,
+    "overrides": [
+      {
+        "telemetryType": "request",
+        "attributes": [
+          {
+            "key": "http.url",
+            "value": ".*/health-check",
+            "matchType": "regexp"
+          }
+        ],
+        "percentage": 10,
+        "id": "filter out health check"
+      },
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "db.statement",
+            "value": "select * from abc",
+            "matchType": "strict"
+          }
+        ],
+        "percentage": 50,
```

Comment:
It may be worth to add a test for the user configuration given in the PR description (0% for the request). I have not seen it.

Reply:
I think it's going to be the same code path as 10% for the request (the key is to test request percentage < dependency percentage < 100%)

Reply:
My suggestion was to be fully sure.

Reply:
added 👍